src/nbl/CMakeLists.txt

```diff
@@ -469,6 +469,18 @@ target_include_directories(Nabla PUBLIC
 	"${THIRD_PARTY_SOURCE_DIR}/dxc/dxc/external/SPIRV-Tools/include"
 	"${THIRD_PARTY_SOURCE_DIR}/dxc/dxc/external/SPIRV-Headers/include"
 )
+
+# nlohmann_json
+set(JSON_BuildTests OFF CACHE INTERNAL "")
```

Comment:
move it [here](https://github.com/Devsh-Graphics-Programming/Nabla/pull/676/files#diff-e286d8c6322ab030a049019a13e4c648f1a84a7e0f51ca0fb82e04e6f902065aR7) and set as ordinary variable, no need to use cache

Reply:
move what where, sorry

Reply:
```cmake
set(JSON_BuildTests OFF)
```

[here](https://github.com/Devsh-Graphics-Programming/Nabla/pull/676/files#diff-e286d8c6322ab030a049019a13e4c648f1a84a7e0f51ca0fb82e04e6f902065aR7) before `add_subdirectory` invocation